app/controllers/posts_controller.rb

```diff
@@ -50,6 +50,21 @@ def interactions
     end
   end
 
+  def mentionable_in_comment
+    respond_to do |format|
+      format.json {
+        if params[:id].present? && params[:q].present?
```

Comment:
@AugierLe42e It looks like there is some misunderstanding.

> I find this counter intuitive to be able to mention people without a @ on comments and not in publisher.

That's not what @SuperTux88 wrote. He wrote

> you should be able to mention the post-author and the last few comment-authors without typing anything (only the @)

So you type the “@” and you will see the post author and the last comment authors immediately.

> no data is available on the front-end right now. There was, not long ago, but #6951 changed that behaviour.

That's wrong. The data you need is the post author and the comment authors. And this information is already available in the front end.